husky_components/matrix/main.js

```diff
@@ -32,7 +32,7 @@ define(function() {
                 this.toggleIcon.bind(this),
                 'tbody > tr > td.value > span[class^="fa-"]'
             );
-            this.sandbox.dom.on(this.$element, 'click', this.toggleRow.bind(this), 'tbody > tr > td:last-child');
+            this.sandbox.dom.on(this.$element, 'click', this.toggleRow.bind(this), 'tbody > tr > td > span:last-of-type');
```

Comment:
Shouldn't it still be `td:last-child`? I think this would also merge the last span of other `td`s